app/views/home/index.html.haml

```diff
@@ -0,0 +1,4 @@
+%h1 Welcome
+
+%form{:action => "/users/sign_up"}
+  %input{:type => "submit", :value => "Sign up"}
```

Comment:
Not sure about this form. Why is it here and why are you not using the form helpers and path_helpers or the devise form? Also, even in this syntax, the form is not complete. You have a Submit button but no input fields for username and emails.

Reply:
This was a misstake, it should just be a link to the sign-up form. I'll fix it.